be/src/vec/functions/array/function_array_with_constant.cpp

```diff
@@ -79,8 +85,29 @@ class FunctionArrayWithConstant : public IFunction {
     }
 };
 
+struct NameArrayWithConstant {
+    static constexpr auto name = "array_with_constant";
+
+    static void get_param_position_from_arguments(const ColumnNumbers& arguments, size_t& num_position,
+                                                  size_t& value_position) {
+        num_position = arguments[0];
+        value_position = arguments[1];
+    }
+};
+
+struct NameArrayRepeat {
+    static constexpr auto name = "array_repeat";
+
+    static void get_param_position_from_arguments(const ColumnNumbers& arguments, size_t& num_position,
+                                                  size_t& value_position) {
+        value_position = arguments[0];
+        num_position = arguments[1];
+    }
+};
+
 void register_function_array_with_constant(SimpleFunctionFactory& factory) {
```

Comment:
better use the `register_alias` to do the work

Reply:
> better use the `register_alias` to do the work

register_alias cannot change param position. 
array_with_constant(n, T) 
array_repeat(T, n)

Reply:
they are not same function because of the params position.
i didn’t find register_alias how to change the params position.
array_with_constant（n，T）
array_repeat（T，n）